charts/fleet-crd/templates/crds.yaml

```diff
@@ -1784,6 +1792,10 @@ spec:
                 type: string
               redeployAgentGeneration:
                 type: integer
+              templateContext:
```

Comment:
I'd prefer it, if the name included "values" somehow, because it is the context for templating helm's `values:`.
Just not `valuesValues`.

Reply:
We could call it `templateValues` ?